### PR TITLE
Added boxing and unboxing; Mapped App.tsx games

### PR DESCRIPTION
**Pull request by AiniusG**

### CODE REVIEW

File: frontend/src/App.tsx
Comment (Dax0s) [inline frontend/src/App.tsx:1]: Smart

File: backend/backend/MemoryGameWithNumbers/Controllers/MemoryGameWithNumbersController.cs
Comment (Aidas21J) [inline backend/backend/MemoryGameWithNumbers/Controllers/MemoryGameWithNumbersController.cs:1]: Sitam faile primete warningu

**Review by Aidas21J — APPROVED** [review]

File: backend/backend/MemoryGameWithNumbers/Controllers/MemoryGameWithNumbersController.cs [hunk 1]
```diff
@@ -1,31 +1,65 @@
...
+    [HttpPost("attempt")]
+    public ActionResult<bool> CheckAttempt([FromBody] List<int?> userAttempt)
+    {
+        if (_correctSequence == null)
+        {
+            return BadRequest("Game not started.");
+        }
 
-            while (gridNumbers.Count < 16)
+        for (int i = 0; i < _correctSequence.Count; i++)
```
Comment (Dax0s) [inline backend/backend/MemoryGameWithNumbers/Controllers/MemoryGameWithNumbersController.cs:42]: Cia galetum naudoti foreach, not sure ar yra skirtumas tarp paprasto for ir foreach

**Review by Dax0s — APPROVED** [review]

File: backend/backend/MemoryGameWithNumbers/Controllers/MemoryGameWithNumbersController.cs (hunk 1 above)
Comment (Aidas21J) [reply backend/backend/MemoryGameWithNumbers/Controllers/MemoryGameWithNumbersController.cs:42]: geriau foreach. both for readability and performance

**Pull request merged**